src/handbook/operations/billing.md

```diff
@@ -30,7 +30,17 @@ on the [quote](/handbook/customer/sales/engagements#generating-a-quote-and-order
 
 ### Creating an invoice
 
-Email billing@flowfuse.com with [this template](https://docs.google.com/document/d/1u9KqAovcKJ8AWpWZhYDEAX90T-BfUfRNlPYKieOKnOY/edit).
+Email billing@flowfuse.com with the following information:
```

Comment:
Why don't we use the template?

Reply:
I use HubSpot templates instead of Google templates, to keep everything consistent. We can link to the HubSpot template too, here's how it looks as a link: https://app-eu1.hubspot.com/templates/26586079/edit/53524964?page=2